routes/rules/jax-rs.yml

```diff
@@ -150,7 +150,8 @@ rules:
     patterns:
       - pattern: |
           @$METHOD
-          $RETURNTYPE $FUNC(...) { ... }
+          @Path($PATH)
```

Comment:
Hmm, I vaguely remember that these `Path` annotations are optional, so this will break some existing findings: https://docs.oracle.com/javaee/6/api/javax/ws/rs/Path.html.

Generally I like the idea of using a consistent metavariables (e.g. `$PATH`, `$METHOD`, etc.) across all languages and frameworks, but I think they have to be optional in terms of detection. Many frameworks have an implicit default in certain cases, and we don't want to miss detecting routes because we're expecting explicit values everywhere.

I'm not sure how best to architect these rules such that we can easily find routes and their optional path, method, etc, information.

Reply:
I think that explains why the tests are failing, probably because the rule I modified is breaking some detections